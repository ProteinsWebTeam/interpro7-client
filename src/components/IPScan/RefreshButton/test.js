// @flow
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { RefreshButton } from '.';

const renderer = new ShallowRenderer();
// const myHeaders = new Headers({'client-cache': true});

describe('<RefreshButton />', () => {
  test('should render', () => {
    renderer.render(<RefreshButton updateJobStatus={() => {}} />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
