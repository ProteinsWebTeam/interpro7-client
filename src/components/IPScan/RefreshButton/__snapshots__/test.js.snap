// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RefreshButton /> should render 1`] = `
<ForwardRef
  aria-label="Manually refresh job information"
  onClick={[Function]}
>
  <Tooltip
    title="Manually refresh job information"
  >
    <span
      className="icon icon-common"
      data-icon="ï‹¹"
    />
  </Tooltip>
</ForwardRef>
`;
